### PR TITLE
move `newUDPEchoServer` to proxier_test.go

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
move userspace proxier test code into test go file 

/kind cleanup

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/proxy/userspace/proxier_test.go [hunk 1]
```diff
@@ -86,6 +86,31 @@ func waitForClosedPortUDP(p *Proxier, proxyPort int) error {
...
+		n, cliAddr, err := r.ReadFrom(buffer[0:])
+		if err != nil {
+			fmt.Printf("ReadFrom failed: %v\n", err)
+			continue
+		}
+		r.WriteTo(buffer[0:n], cliAddr)
+	}
+}
+
+func newUDPEchoServer() (*udpEchoServer, error) {
```
Comment (m1093782566) [inline pkg/proxy/userspace/proxier_test.go]: nit picking: move this function below udpEchoServer struct? So, the display looks like:

1. type define

2. Initialization

3. functions

Comment (hzxuzhonghu) [reply pkg/proxy/userspace/proxier_test.go]: make sense

Comment (hzxuzhonghu) [reply pkg/proxy/userspace/proxier_test.go]: done

**Comment by m1093782566** [comment]
Please squash 1st and 2nd commit. Then lgtm

**Pull request merged**